UiPharmecy/.idea/modules.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/UiPharmecy.iml" filepath="$PROJECT_DIR$/UiPharmecy.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
[Same](https://github.com/AhsanBracu/Image-To-Pdf/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R42)